root_VS2012/programs/C#/Frameworks/Infrastructure/Public/Resources/PublicExceptionMessageResource.resx

```diff
@@ -180,6 +180,9 @@
   <data name="NOT_IMPLEMENTED" xml:space="preserve">
     <value>This processing is not Implemented.</value>
   </data>
+  <data name="NOT_SUPPORTED" xml:space="preserve">
+    <value>This process is not supported .</value>
```

Comment:
@daisukenishino 

There is a space character between "supported" and "."
Please modify as follows.

```xml
<value>This process is not supported.</value>
```